packages/forms/src/elements/currency/currency.mdx

```diff
@@ -211,18 +211,19 @@ Accepted config props: FlexProps, SpaceProps
-| testId         | false    | string   | data attribute for testers                                                                               |
+| Property       | Required | Type     | Description                                                                                                                           |
+| -------------- | -------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| after          | false    | string                                | emulates text passed to ::after pseudo-selector                                                          |
+| before         | false    | string                                | emulates text passed to ::before pseudo-selector                                                         |
+| callback       | false    | function                              | callback function to be executed after onChange, receives the input value as a number                    |
+| cfg            | false    | object                                | FlexProps & SpaceProps                                                                                   |
+| decimalPlaces  | false    | number                                | the number of decimal places used for formatting the value, default is 2                                 |
+| disabled       | false    | boolean                               | Disable input field                                                                                      |
+| hint           | false    | string                                | More detailed description about the field                                                                |
+| initialValue   | false    | number                                | Initial value for the input, will be automatically formatted                                             |
+| label          | true     | string                                | Input description                                                                                        |
+| maxInputLength | false    | number                                | the max length for the input (including ',' and '.'), default is 16 + decimalPlaces (999,999,999,999.99) |
+| noLeftBorder   | false    | boolean                               | disables the left border when detecting error                                                            |
+| optionalText   | false    | boolean                               | allows hiding "optional" text when field is not required                                                 |
+| testId         | false    | string                                | data attribute for testers                                                                               |
+| i18n           | false    | FieldWithAriaLabelExtenstionI18nProps | allows overridding of text in currency input                                                             |
```

Comment:
You could add documentation for `aria-label` too. Although it's not a prop, it is an expected attribute just like the props and it would be useful to clarify `aria-label` vs `label`.

Reply:
I've added descriptions of the aria-label and have added example components to the currency and number mdx files showing the uses.